app/src/main/java/org/oppia/app/topic/conceptcard/ConceptCardViewModel.kt

```diff
@@ -0,0 +1,9 @@
+package org.oppia.app.topic.conceptcard
+
+import androidx.lifecycle.ViewModel
+import org.oppia.app.fragment.FragmentScope
+import javax.inject.Inject
+
+@FragmentScope
+class ConceptCardViewModel @Inject constructor() : ViewModel() {
```

Comment:
Consider putting a value in here to bind with in the meantime so that you have DataBinding set up, and provide yourself the basis for filling in the rest of the concept card once the API is finished.

Reply:
Also: please add documentation.